.github/workflows/ci.yml

```
@@ -0,0 +1,150 @@
+  contents: read
+  actions: read
+  pull-requests: write
+  security-events: write
+
+jobs:
+  lint-and-type-check:
+    runs-on: ubuntu-latest
+    name: Lint & Type Safety
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
```

actionlint (1.7.10)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)